app/views/admin/configs/show.html.erb

```diff
@@ -0,0 +1,1306 @@
+<div class="grid gap-6" data-controller="config">
```

Comment:
Is this file supposed to be here? I think this is a mistake in merge conflict because it was renamed to `app/views/admin/settings/show.html.erb `

Reply:
Great catch @maestromac it was definitely a mistake in a merge conflict resolution, thanks for mentioning this!